float64[] q
float64[] qd
float64[] qdd
float64[] Fext
---
float64[] Tau
float64 KE